### DISCUSSION TITLE
I'm getting Goggle Chrome related icons in my Applications menu. Who's responsible? Elementary or Google?

### DISCUSSION THREAD

**Discussion by jksgvb** [elementary/applications-menu#592] [Q&A] [answered]
Also, I want to get rid of them. How do I get rid of these unauthorized installations in my Applications menu?

**Comment by ryonakano** [comment, marked as answer]
I also faced this "issue" before. Icons of "YouTube" or "Google Drive" which I didn't installed explicitly are shown in Applications Menu.

> I'm getting Goggle Chrome related icons in my Applications menu. Who's responsible? Elementary or Google?

They should be installed with Google Chrome since elementary is not related to Google.

> Also, I want to get rid of them. How do I get rid of these unauthorized installations in my Applications menu?

These icons are displayed by some .desktop files under `~/.local/share/applications/`.

**Reply by jksgvb** [reply]
Much thanks ryonakano. Deleted all. If I get rid of chrome would chromium be just as bad? I'm in the process of transferring bookmarks, passwords, etc to Floorp which I like but I still like chrome for email access. Open to suggestions here on how to stop the evil.

**Reply by teamcons** [reply]
> Much thanks ryonakano. Deleted all. If I get rid of chrome would chromium be just as bad? I'm in the process of transferring bookmarks, passwords, etc to Floorp which I like but I still like chrome for email access. Open to suggestions here on how to stop the evil.

Probably simply edit said .desktop files with the relevant FreeDesktop variable for hiding it in menus
They will still exist, but wont clutter around
theres a few appmenu editors in appcenter that can be used for this, if you fancy a gui

**Comment by jksgvb** [comment]
I got rid of chrome and re-installed the flatpak version. Hopefully that will keep it better coralled. When I went to google sheets, chrome complained that it couldn't create handy buttons for me in the ~/.local/share/applications/ folder. That will have to do for now till I can get another email system going.
